src/components/wizard.component.ts

```diff
@@ -42,7 +45,8 @@ import {NavigationMode} from '../navigation/navigation-mode.interface';
   selector: 'wizard',
   templateUrl: 'wizard.component.html',
   styleUrls: ['wizard.component.less'],
-  providers: [WizardState]
+  providers: [WizardState],
+  host: {'class': 'ng2-archwizard'}
```

Comment:
Do we absolutely need to add a (in my eyes) redundant `ng2-archwizard` class to the `wizard` component?
I mean the component is already called `wizard` and should therefore be accessible via `wizard` in css, right?

This comment doesn't mean, that we mustn't add a `ng2-archwizard` css class to the component, I would just like to discuss the pros and cons, before we add it, because afterwards it may be hard to remove it again, if we decide later on that it is unnecessary.